preprocessing_scripts/ert_rft.py

```diff
@@ -0,0 +1,196 @@
+import os
+import glob
+
+import pandas as pd
+from fmu.ensemble import ScratchEnsemble, EnsembleSet
+
+######################################################################
+# df_rft_obs: this function will make a dataframe of ERT rft obsdata
+# step 1
+######################################################################
+
+
+def get_rft_reportsteps(reportsteps):
+    """Combine rft data from multiple report steps"""
+    return pd.concat(
+        [
+            df_rft_obs(rs["folder"], rs["well_date"], rs["reportstep"])
+            for rs in reportsteps
+        ]
+    )
+
+
+def df_rft_obs(obsfolder, well_date_rft="well_date_rft.txt", reportstep=1):
+    """Extract observed RFT data from ERT inputfolder structure """
+    rft_wellname = []
+    rft_date = []
+    rft_meas = []
+    rft_meas_err = []
+    rft_east = []
+    rft_north = []
+    rft_md = []
+    rft_tvd = []
+    rft_zone = []
+
+    with open(os.path.join(obsfolder, well_date_rft)) as f:
+        for count, line in enumerate(f):
+
+            try:
+                items = line.split()
+
+                if str(items[4]) != str(reportstep):
+
+                    continue
+                well = items[0]
+                date = items[3] + items[2] + items[1]
+                obsfile = os.path.join(obsfolder, well + ".obs")
+                txtfile = os.path.join(obsfolder, well + ".txt")
+            except:
+                print("skipping line", count, "in file:", well_date_rft)
+                continue
+
+            with open(obsfile) as f_obs:
+                for count1, obs in enumerate(f_obs):
+                    try:
+                        obs_items = obs.split()
+                        rft_meas.append(obs_items[0])
+                        rft_meas_err.append(obs_items[1])
+                        rft_wellname.append(well)
+                        rft_date.append(date)
+                    except:
+                        print("skipping line", count1, "in file:", obsfile)
+                        continue
+
+            with open(txtfile) as f_txt:
+                for count2, txt in enumerate(f_txt):
+                    try:
+                        txt_items = txt.split()
+                        rft_east.append(txt_items[0])
+                        rft_north.append(txt_items[1])
+                        rft_md.append(txt_items[2])
+                        rft_tvd.append(txt_items[3])
+                        try:
+                            rft_zone.append(txt_items[4])
+                        except:
+                            rft_zone.append(None)
+                    except:
+                        print("skipping line", count2, "in file:", txtfile)
+                        continue
```

Comment:
Is this solution really safe against different vector lengths, and mixing of data if e.g. obsfile exists and txtfile doesn't? 
Think we should try to build a full rft data point (one value for each vector), and only append if all are ok?